extension/linkify-urls-in-code.js

```diff
@@ -27,11 +27,13 @@ window.linkifyURLsInCode = (() => {
 		$(codeBlobs)
 		.toArray()
 		.forEach(blob => {
-			const blobHTML = blob.innerHTML;
-			if (hasURL(blobHTML)) {
-				// Match URLs and remove < or > from beginning or end
-				const URLmatch = blobHTML.match(URLRegex)[0].replace(/(^&lt)|(&gt$)/, '');
-				blob.innerHTML = blobHTML.replace(URLmatch, linkifyURL(URLmatch));
+			if (hasURL(blob.innerHTML)) {
+				const matches = blob.innerHTML.match(getURLRegex());
+				for (let match of matches) {
+					// Remove < or > from beginning or end of an URL
+					match = match.replace(/(^&lt)|(&gt$)/, '');
+					blob.innerHTML = blob.innerHTML.replace(match, linkifyURL(match));
```

Comment:
We cannot call `replace` on a "cached" version of blob.innerHTML due to changes we’re making here in the loop. By using `blobHTML` instead of `blob.innerHTML` we will _linkify_ only the last link in each blob. That was a reason why I removed the `const` from a few lines above in the first place.